### PR TITLE
Create user and role in postgres and connect node to db via sequelize

### PR DESCRIPTION
**Pull request by gldmtr**

### CODE REVIEW

File: cm/vagrant.yml [hunk 1]
```diff
@@ -12,5 +12,22 @@
...
+          with_items:
+          - libpq-dev
+          - python-psycopg2
+          - postgresql-9.5
+        - service: name=postgresql state=started
+        - blockinfile:
+            dest: /etc/profile.d/interviews_db.sh
+            create: yes
+            block: |
+              DATABASE_URL=postgresql://vagrant:vagrant@localhost/interviews
```
Comment (Shamash2014) [inline cm/vagrant.yml:25]: Maybe get database url from ENV?

Comment (gldmtr) [inline cm/vagrant.yml:25]: @Shamash2014 Can you clarify, what do you mean?

Comment (Shamash2014) [inline cm/vagrant.yml:25]: My point: we can obtain postgres url from process.env.POSTGRES_URL or whatever, defined as environment variable in the system. We also can use process.env.URL || default value

Comment (gldmtr) [inline cm/vagrant.yml:25]: @Shamash2014 heroku defines DATABASE_URL, and we should use it. In this ansible task I export this var for using it in development env.

**Comment by sashashakun** [comment]
Due #50

**Comment by sashashakun** [comment]
I think we can merge it.

**Comment by sashashakun** [comment]
I will merge, but don't close issue

**Pull request merged**